docs/topics_5/web-custom-rules.adoc

```diff
@@ -1,51 +1,78 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='using_custom_rules_and_labels_{context}']
-= Using custom rules and labels
+= Using global custom rules and labels
 
```

Comment:
I think this module might be obsolete. Please check whether it is used. If not, delete it.